main_test.go

```diff
@@ -3,21 +3,31 @@ package main
 import (
 	"errors"
 	"fmt"
+	"io/ioutil"
+	"log"
 	"os"
 	"testing"
 
+	"github.com/Sirupsen/logrus"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ecr"
 	"github.com/stretchr/testify/assert"
 	"github.com/upmc-enterprises/registry-creds/k8sutil"
 	"golang.org/x/net/context"
 	"golang.org/x/oauth2"
 	coreType "k8s.io/client-go/kubernetes/typed/core/v1"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	v1fake "k8s.io/client-go/kubernetes/typed/core/v1/fake"
 	"k8s.io/client-go/pkg/api"
 	"k8s.io/client-go/pkg/api/v1"
 	"k8s.io/client-go/pkg/watch"
 )
 
+func init() {
+	log.SetOutput(ioutil.Discard)
+	logrus.SetOutput(ioutil.Discard)
+}
```

Comment:
this will set all log output to /dev/null during tests so as to not clutter the screen.